state/state.go

```diff
@@ -1105,12 +1105,16 @@ func (st *State) addPeerRelationsOps(applicationname string, peers map[string]ch
 			ModelUUID: st.ModelUUID(),
 			Updated:   now.UnixNano(),
 		}
-		ops = append(ops, txn.Op{
-			C:      relationsC,
-			Id:     relDoc.DocID,
-			Assert: txn.DocMissing,
-			Insert: relDoc,
-		}, createStatusOp(st, relationGlobalScope(relId), relationStatusDoc))
+		ops = append(ops,
+			txn.Op{
+				C:      relationsC,
+				Id:     relDoc.DocID,
+				Assert: txn.DocMissing,
+				Insert: relDoc,
+			},
+			createStatusOp(st, relationGlobalScope(relId), relationStatusDoc),
+			createSettingsOp(settingsC, relationApplicationSettingsKey(relId, eps[0]), nil),
```

Comment:
instead of nil,
emptySettings := make(map[string]interface{})
createSettingsOp(settingsC, relationApplicationSettingsKey(relId, eps[0]), emptySettings)

Reply:
Changed.